cudd/cuddUtil.c

```diff
@@ -1453,6 +1511,52 @@ Cudd_bddPickOneMinterm(
 
 }  /* end of Cudd_bddPickOneMinterm */
 
+/**
+  @brief Picks one on-set minterm from the given %DD.
+
+  @details The minterm is represented as a decision diagram. Whenever there is
+  a choice for a variable (i.e., neither of the children is the 0-terminal), the
+  value taken from the cube <code>choice</code>, or it is false if
+  <code>choice</code> is independent of that variable. If <code>f</code> is the
+  constant function 0 (i.e., there is no cube), this function simply returns the
+  0-terminal.
+
+  @return a pointer to the %BDD for the minterm if successful; NULL if the
+  procedure ran out of memory.
+
+  @sideeffect None
+
+  @see Cudd_bddPickOneCube Cudd_bddPickOneMinterm
+
+*/
+DdNode *
+Cudd_addPickOneMintermSet(
+  DdManager * dd /**< manager */,
+  DdNode * f /**< function from which to pick one minterm */,
+  DdNode * choice /**< cube describing the choice */)
```

Comment:
I suppose with the *Set* suffix you want to differentiate it from `Cudd_bddPickOneMinterm` where the set of variables *vars* is provided as a mere array of integers?

Reply:
Yes, indeed.